src/work-allocation-2/enums/info-message.ts

```diff
@@ -3,7 +3,7 @@ export enum InfoMessage {
   ASSIGNED_TASK_AVAILABLE_IN_MY_TASKS = 'You\'ve assigned yourself a task. It\'s available in My tasks.',
   CANCELLED_TASK = 'You\'ve cancelled a task.',
   LIST_OF_TASKS_REFRESHED = 'The list has been refreshed.',
-  MARKED_TASK_AS_DONE = 'You\'ve marked a task as done.',
+  MARKED_TASK_AS_DONE = 'You\'ve marked a task as done. It has been removed from the task list.',
```

Comment:
This can be changed to use backticks as it will look cleaner and save having you having to escape out the single quote inside the string.

Reply:
**nit pick**

Reply:
Tslint doesn't like back ticks on enums